symphony-bdk-examples/simple-hello-world/src/main/java/com/symphony/bdk/examples/HelloWorldMain.java

```diff
@@ -0,0 +1,64 @@
+package com.symphony.bdk.examples;
+
+import com.symphony.bdk.core.api.invoker.ApiClient;
+import com.symphony.bdk.core.api.invoker.jersey2.ApiClientJersey2;
+import com.symphony.bdk.core.auth.JwtHelper;
+import com.symphony.bdk.gen.api.AuthenticationApi;
+import com.symphony.bdk.gen.api.MessageApi;
+import com.symphony.bdk.gen.api.MessagesApi;
+import com.symphony.bdk.gen.api.model.AuthenticateRequest;
+import com.symphony.bdk.gen.api.model.V4Message;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.IOUtils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.security.PrivateKey;
+
+@Slf4j
+public class HelloWorldMain {
+
+  private static final String POD_BASE_URL = "https://devx1.symphony.com";
+
+  public static void main(String[] args) throws Exception {
+
+    final AuthenticationApi loginAuthApi = new AuthenticationApi(new ApiClientJersey2(POD_BASE_URL + "/login"));
+    final AuthenticationApi kmAuthApi = new AuthenticationApi(new ApiClientJersey2(POD_BASE_URL + "/relay"));
+
+    final AuthenticateRequest request = new AuthenticateRequest();
+    request.setToken(generateJwt());
+
+    final String sessionToken = loginAuthApi.pubkeyAuthenticatePost(request).getToken();
```

Comment:
at the Java API level do we need the HTTP method to appear (post)?

Reply:
Nope, this class is just a _temporary_ hello world, we will hide those low-level clients in cleaner services